### DISCUSSION TITLE
how can i login to my brand account using spmp desktop client?

### DISCUSSION THREAD

**Discussion by anispwyn** [sayaka-sh/spmp#239] [Q&A] [answered]
i know that you can login to your account via cookies, but how can i get the brand account cookies?

**Comment by sayaka-sh** [comment, marked as answer]
You should be able to select a brand account after inputting the headers now (a15a5cd). Shouldn't matter which headers you input.

You can grab the build from the [actions page](https://github.com/toasterofbread/spmp/actions).
